src/components/button/AposButton.vue

```diff
@@ -2,7 +2,7 @@
   <button type="button" @click="click" class="apos-button" :class="modifierClass" v-bind:busy="busy" v-bind:disabled="isDisabled">
     <AposLoading />
     <div class="apos-button__content">
-      <component :size="16" class="apos-button__icon" v-if="icon" v-bind:is="iconComponent"></component>
+      <component :size="15" class="apos-button__icon" v-if="icon" v-bind:is="iconComponent"></component>
       <span class="apos-button__label">
         {{ label }}
```

Comment:
Is label a required prop? If not maybe, this should be `v-if`? For accessibility purposes, might always want to require a label but have a prop that allows you to hide the text with a `sr-only` class.